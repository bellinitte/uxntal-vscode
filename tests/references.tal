.label .&sublabel
,label ,&sublabel
;label ;&sublabel
:label :&sublabel
-label -&sublabel
=label =&sublabel
_label _&sublabel

label-label
label-la/bel ( `/` highlighted )
la/bel-la/bel ( Second `/` not highlighted )
/
//

.label-label
.label-la/bel ( `/` highlighted )
.la/bel-la/bel ( Second `/` not highlighted )
.
./
.//

.&sublabel-sublabel
.&sublabel-subla/bel ( `/` not highlighted )
.&
.&/
