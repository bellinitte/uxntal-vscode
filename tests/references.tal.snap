>.label .&sublabel
#^ source.tal punctuation.rune.uxntal
# ^^^^^ source.tal entity.name.function.uxntal
#      ^ source.tal
#       ^ source.tal punctuation.rune.uxntal
#        ^ source.tal punctuation.rune.uxntal
#         ^^^^^^^^ source.tal entity.name.function.uxntal
>,label ,&sublabel
#^ source.tal punctuation.rune.uxntal
# ^^^^^ source.tal entity.name.function.uxntal
#      ^ source.tal
#       ^ source.tal punctuation.rune.uxntal
#        ^ source.tal punctuation.rune.uxntal
#         ^^^^^^^^ source.tal entity.name.function.uxntal
>;label ;&sublabel
#^ source.tal punctuation.rune.uxntal
# ^^^^^ source.tal entity.name.function.uxntal
#      ^ source.tal
#       ^ source.tal punctuation.rune.uxntal
#        ^ source.tal punctuation.rune.uxntal
#         ^^^^^^^^ source.tal entity.name.function.uxntal
>:label :&sublabel
#^ source.tal punctuation.rune.uxntal
# ^^^^^ source.tal entity.name.function.uxntal
#      ^ source.tal
#       ^ source.tal punctuation.rune.uxntal
#        ^ source.tal punctuation.rune.uxntal
#         ^^^^^^^^ source.tal entity.name.function.uxntal
>-label -&sublabel
#^ source.tal punctuation.rune.uxntal
# ^^^^^ source.tal entity.name.function.uxntal
#      ^ source.tal
#       ^ source.tal punctuation.rune.uxntal
#        ^ source.tal punctuation.rune.uxntal
#         ^^^^^^^^ source.tal entity.name.function.uxntal
>=label =&sublabel
#^ source.tal punctuation.rune.uxntal
# ^^^^^ source.tal entity.name.function.uxntal
#      ^ source.tal
#       ^ source.tal punctuation.rune.uxntal
#        ^ source.tal punctuation.rune.uxntal
#         ^^^^^^^^ source.tal entity.name.function.uxntal
>_label _&sublabel
#^ source.tal punctuation.rune.uxntal
# ^^^^^ source.tal entity.name.function.uxntal
#      ^ source.tal
#       ^ source.tal punctuation.rune.uxntal
#        ^ source.tal punctuation.rune.uxntal
#         ^^^^^^^^ source.tal entity.name.function.uxntal
>!label !&sublabel
#^ source.tal punctuation.rune.uxntal
# ^^^^^ source.tal entity.name.function.uxntal
#      ^ source.tal
#       ^ source.tal punctuation.rune.uxntal
#        ^ source.tal punctuation.rune.uxntal
#         ^^^^^^^^ source.tal entity.name.function.uxntal
>?label ?&sublabel
#^ source.tal punctuation.rune.uxntal
# ^^^^^ source.tal entity.name.function.uxntal
#      ^ source.tal
#       ^ source.tal punctuation.rune.uxntal
#        ^ source.tal punctuation.rune.uxntal
#         ^^^^^^^^ source.tal entity.name.function.uxntal
>
>label-label
#^^^^^^^^^^^ source.tal entity.name.function.uxntal
>label-la/bel ( `/` highlighted )
#^^^^^^^^ source.tal entity.name.function.uxntal
#        ^ source.tal punctuation.rune.uxntal
#         ^^^ source.tal entity.name.function.uxntal
#            ^ source.tal
#             ^ source.tal comment.uxntal
#              ^ source.tal
#               ^^^ source.tal comment.uxntal
#                  ^ source.tal
#                   ^^^^^^^^^^^ source.tal comment.uxntal
#                              ^ source.tal
#                               ^ source.tal comment.uxntal
>la/bel-la/bel ( Second `/` not highlighted )
#^^ source.tal entity.name.function.uxntal
#  ^ source.tal punctuation.rune.uxntal
#   ^^^^^^^^^^ source.tal entity.name.function.uxntal
#             ^ source.tal
#              ^ source.tal comment.uxntal
#               ^ source.tal
#                ^^^^^^ source.tal comment.uxntal
#                      ^ source.tal
#                       ^^^ source.tal comment.uxntal
#                          ^ source.tal
#                           ^^^ source.tal comment.uxntal
#                              ^ source.tal
#                               ^^^^^^^^^^^ source.tal comment.uxntal
#                                          ^ source.tal
#                                           ^ source.tal comment.uxntal
>/
#^ source.tal punctuation.rune.uxntal
>//
#^ source.tal punctuation.rune.uxntal
# ^ source.tal entity.name.function.uxntal
>
>.label-label
#^ source.tal punctuation.rune.uxntal
# ^^^^^^^^^^^ source.tal entity.name.function.uxntal
>.label-la/bel ( `/` highlighted )
#^ source.tal punctuation.rune.uxntal
# ^^^^^^^^ source.tal entity.name.function.uxntal
#         ^ source.tal punctuation.rune.uxntal
#          ^^^ source.tal entity.name.function.uxntal
#             ^ source.tal
#              ^ source.tal comment.uxntal
#               ^ source.tal
#                ^^^ source.tal comment.uxntal
#                   ^ source.tal
#                    ^^^^^^^^^^^ source.tal comment.uxntal
#                               ^ source.tal
#                                ^ source.tal comment.uxntal
>.la/bel-la/bel ( Second `/` not highlighted )
#^ source.tal punctuation.rune.uxntal
# ^^ source.tal entity.name.function.uxntal
#   ^ source.tal punctuation.rune.uxntal
#    ^^^^^^^^^^ source.tal entity.name.function.uxntal
#              ^ source.tal
#               ^ source.tal comment.uxntal
#                ^ source.tal
#                 ^^^^^^ source.tal comment.uxntal
#                       ^ source.tal
#                        ^^^ source.tal comment.uxntal
#                           ^ source.tal
#                            ^^^ source.tal comment.uxntal
#                               ^ source.tal
#                                ^^^^^^^^^^^ source.tal comment.uxntal
#                                           ^ source.tal
#                                            ^ source.tal comment.uxntal
>.
#^ source.tal punctuation.rune.uxntal
>./
#^ source.tal punctuation.rune.uxntal
# ^ source.tal punctuation.rune.uxntal
>.//
#^ source.tal punctuation.rune.uxntal
# ^ source.tal punctuation.rune.uxntal
#  ^ source.tal entity.name.function.uxntal
>
>.&sublabel-sublabel
#^ source.tal punctuation.rune.uxntal
# ^ source.tal punctuation.rune.uxntal
#  ^^^^^^^^^^^^^^^^^ source.tal entity.name.function.uxntal
>.&sublabel-subla/bel ( `/` not highlighted )
#^ source.tal punctuation.rune.uxntal
# ^ source.tal punctuation.rune.uxntal
#  ^^^^^^^^^^^^^^^^^^ source.tal entity.name.function.uxntal
#                    ^ source.tal
#                     ^ source.tal comment.uxntal
#                      ^ source.tal
#                       ^^^ source.tal comment.uxntal
#                          ^ source.tal
#                           ^^^ source.tal comment.uxntal
#                              ^ source.tal
#                               ^^^^^^^^^^^ source.tal comment.uxntal
#                                          ^ source.tal
#                                           ^ source.tal comment.uxntal
>.&
#^ source.tal punctuation.rune.uxntal
# ^ source.tal punctuation.rune.uxntal
>.&/
#^ source.tal punctuation.rune.uxntal
# ^ source.tal punctuation.rune.uxntal
#  ^ source.tal entity.name.function.uxntal
>